doc/userdoc/release_notes/v3.1/index.rst

```diff
@@ -0,0 +1,62 @@
+All about NEST 3.1
+==================
+
+This page contains a summary of all breaking and non-breaking changes
+from NEST 3.0 to NEST 3.1. In addition to the `auto-generated release
+notes on GitHub <https://github.com/nest/nest-simulator/releases/>`_,
+this page also contains transition information that helps you to
+update your simulation scripts when you come from an older version of
+NEST.
+
+If you transition from a version earlier than 3.0, please see our
+extensive :doc:`transition guide from NEST 2.x to 3.0
+<nest2_to_nest3/index>`.
+
+.. contents:: On this page you'll find
+   :local:
+   :depth: 1
+
+
+Access to kernel properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With NEST 3.1 it is easier than ever to access properties of the NEST
+simulation kernel. Instead of using the traditional access methods
+:py:func:`.GetKernelStatus` and :py:func:`.SetKernelStatus`,
+properties can now also be set and retrieved via *direct kernel
+attributes*.
+
+Where you previously had ``nest.SetKernelStatus({"resolution": 0.2})``
+in your simulation script, you can now just write ``nest.resolution =
+0.5``. Kernel attributes are first-class citizens of the NEST Python
```

Comment:
Is "first-class citizen" a term that's used in Python?

It seems a bit sketchy to me.

Reply:
Some might even say you're not a real Python user until you've complained not being a first-class citizen somewhere  😂 

Maybe:

```
Where you previously had ``nest.SetKernelStatus({"resolution": 0.2})``
in your simulation script, you can now write ``nest.resolution = 0.5``. 
Kernel attributes are documented and can make use of tab completion.
```

Reply:
@jougs My suggestion would simply be to cut it so the sentence would read something like "Kernel attributes now ome with their own docstrings and even tab-completion works for them!"